salt/transport/zeromq.py

```diff
@@ -0,0 +1,636 @@
+# -*- coding: utf-8 -*-
+'''
+Zeromq transport classes
+'''
+
+import os
+import errno
+import hashlib
+
+from random import randint
+
+# Import Salt Libs
+import salt.auth
+import salt.crypt
+import salt.utils
+import salt.utils.verify
+import salt.utils.event
+import salt.payload
+import logging
+
+
+import salt.transport.client
+import salt.transport.server
+import salt.transport.mixins.auth
+from salt.exceptions import SaltReqTimeoutError
+
+import zmq
+import zmq.eventloop.ioloop
+import zmq.eventloop.zmqstream
+
+# tornado imports
+import tornado
+import tornado.tcpserver
```

Comment:
Are the tornado.tcpserver and tornado.tcpclient imports needed in this module? I don't see them used.